etc/kayobe/inventory/group_vars/seed/network-interfaces

```diff
@@ -30,6 +30,9 @@ storage_mgmt_net_interface: "{{ provision_oc_net_interface }}.{{ storage_mgmt_ne
 # storage_mgmt_net_bridge_ports:
 # storage_mgmt_net_bond_slaves:
 
+# Admin backdoor to seed node
+ilab_net_interface: "{{ ilab_net_interface }}.{{ ilab_net_vlan }}"
```

Comment:
should this be provision_oc_net_interface?

Reply:
Ah, I see, so we use this network to provision the seed. I should have picked up on that..

Reply:
Good spot, avoided infinite recursion, thanks!